com.opera.Opera.yaml

```
@@ -0,0 +1,116 @@
+app-id: com.opera.Opera
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.chromium.Chromium.BaseApp
+base-version: '22.08'
+command: opera
+separate-locales: false
+finish-args:
+  - --require-version=1.8.2
+  - --device=all
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --share=ipc
+  - --share=network
+  - --socket=cups
+  - --socket=pcsc # FIDO2
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --socket=wayland
```

```suggestion
  - --socket=x11
```

It doesn't work on Wayland.

```
❱ flatpak run com.opera.Opera
[2:2:0104/100320.696748:ERROR:ozone_platform_x11.cc(238)] Missing X server or $DISPLAY
[2:2:0104/100320.696779:ERROR:env.cc(255)] The platform failed to initialize.  Exiting.
```